common/authorization/interceptor.go

```diff
@@ -121,24 +124,28 @@ func (a *interceptor) Interceptor(
 
 		scope := a.getMetricsScope(metrics.AuthorizationScope, namespace)
 		sw := scope.StartTimer(metrics.ServiceAuthorizationLatency)
-		defer sw.Stop()
```

Comment:
why removing the defer?
if the intention is to have a accurate authorization latency measurement, then turn the block below into a func and use defer
```go
if a.authorizer != nil {
    ...
}
```

Reply:
It was one bug and I introduced another. Before `handler` call was included in `ServiceAuthorizationLatency` and now timer won't stop in case if auth failed. Fixing it.